posit-dev/positron
# Fix data explorer layout icons

### Intent

Addresses #2877.

<img width="184" alt="image" src="https://github.com/posit-dev/positron/assets/470418/9ea962ab-889f-4d59-b34b-abec4ad7cdde">

### Approach

There were two problems here:

- The first, and most obvious, was that the icons were renamed from `data-tool-*` to `data-explorer-*` on the Positron side, but not on the codicons side. This change addresses that by renaming the codicons (https://github.com/posit-dev/positron-codicons/commit/e8e512f4e8f79fdba58f3c6f823071c2ef597632) so they match the names in the UI.
- Separately, it looks like we never hooked up the UI so that the icon updates when we change the layout, so I added that feature. 

### QA Notes

The icon should now update as you switch layouts, so e.g. if you swap to a columns-on-the-right layout, the icon should adjust to one showing columns on the right.
